### PR TITLE
fixed mathcheck export value

### PR DESCRIPTION
**Pull request by tom-wa**

### CODE REVIEW

File: src/plugins/mathcheck/mathcheck.c [hunk 1]
```diff
@@ -21,8 +21,8 @@
 #include "mathcheck.h"
 
 #define MIN_VALID_STACK 3
-#define EPSILON 0.00001
-
+#define EPSILON 0.00001f
```
Comment (markus2330) [inline src/plugins/mathcheck/mathcheck.c]: Why not simply use `"0.00001"` (or put quotes using two macros)?

**Comment by markus2330** [comment]
Thanks for quick response!

- Please also add a testcase for a wrong and correct calculation.
- Please also add fix #327 in the comment of the commit.

File: src/plugins/mathcheck/mathcheck.c (hunk 1 above)
Comment (tom-wa) [inline src/plugins/mathcheck/mathcheck.c]: had a lot of trouble decidin

Comment (markus2330) [inline src/plugins/mathcheck/mathcheck.c]: https://gcc.gnu.org/onlinedocs/cpp/Stringification.html

**Comment by markus2330** [comment]
I meant the git commit of git commit, but never mind ;)

Thanks for the fix!

**Pull request merged**